### PR TITLE
fix: Notification error toastr do not disappear 

### PR DESCRIPTION
**Pull request by kushthedude**
- The banner doesn't disappear & doesn't have any cross button
- It appears when an adapter is already configured
- The settings link looks too much odd.
- It does not differentiate b/w configures & missing adapter

here is the screenshot when I ping my istio adapter which is already configured from the dashboard
Screenshot 2020-08-05 at 5 45 43 PM

### CODE REVIEW

**Comment by kushthedude** [comment]
Due to time limitations, I have just reverted the fix rather than enhancing the toastr

**Comment by Nikhil-Ladha** [comment]
I found the errors that are in the code, the points 1, 2 can be taken care of. The 3rd point for the Settings button, I think the button was made that way intentionally. 4. The function only checks whether the adapter can be ping led or not, so it will not differentiate between missing and configuring the adapter.

**Comment by kushthedude** [comment]
> The function only checks whether the adapter can be ping led or not, so it will not differentiate between missing and configuring the adapter.

So you mean whether my adapter is configured or not, I need to see this error message

**Comment by kushthedude** [comment]
> The 3rd point for the Settings button, I think the button was made that way intentionally

That's too much odd, a simple anchor tag would be way more beautiful

**Comment by Nikhil-Ladha** [comment]
> > The function only checks whether the adapter can be ping led or not, so it will not differentiate between missing and configuring the adapter.
> 
> So you mean whether my adapter is configured or not, I need to see this error message

If your adapter in configured, but it's not reachable only then you will see this message.

**Comment by Nikhil-Ladha** [comment]
> > The 3rd point for the Settings button, I think the button was made that way intentionally
> 
> That's too much odd, a simple anchor tag would be way more beautiful

Agreed, will make some changes to that.

**Comment by Nikhil-Ladha** [comment]
Don't merge this PR, I will push changes here itself.

**Comment by kushthedude** [comment]
> Don't merge this PR, I will push changes here itself.

Sure, Please

**Comment by Nikhil-Ladha** [comment]
Now, it's good to go.

**Comment by kushthedude** [comment]
> Now, it's good to go.

Just reduced the timeout, else looks good!

File: ui/components/DashboardComponent.js [hunk 1]
```diff
@@ -81,7 +81,6 @@ const styles = (theme) => ({
   },
   redirectButton: {
     marginLeft: '-.5em',
-    backgroundColor: '#ccc',
```
Comment (leecalcote) [inline ui/components/DashboardComponent.js:84]: A slightly off color from the exist red notification will draw attention, but perhaps, avoid the jarring affect that this light gray induced.

What will the "Settings" link/button look like now?

Comment (Nikhil-Ladha) [reply ui/components/DashboardComponent.js:84]: It will be in ' Black' color, and on hovering a bit of dark red background color will come up.

**Pull request merged**